pkg/node/volumemgr.go

```diff
@@ -222,6 +222,21 @@ func (m *VolumeManager) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 			ll.Debug("Change volume status from Created to Removing")
 		case apiV1.Removing:
 		case apiV1.Removed:
+			// we need to update annotation on related drive CRD
+			// todo can we do polling instead?
+			ll.Infof("Volume %s is removed. Updating related", volume.Name)
+			drive := m.crHelper.GetDriveCRByUUID(volume.Spec.Location)
+			annotationKey := fmt.Sprintf("volume/%s", volume.Name)
+			// init map if empty
+			if drive.Annotations == nil {
+				drive.Annotations = make(map[string]string)
+			}
+			drive.Annotations[annotationKey] = apiV1.Removed
```

Comment:
Why do we need to annotate drive CR?

Reply:
To trigger reconcile method for this drive instance.